### DISCUSSION TITLE
Redirecting ports for the caddy to non-default ones causes the certificate generation to fail?

### DISCUSSION THREAD

**Discussion by bartoszbielawski** [dec0dOS/zero-ui#36] [Q&A] [answered]
Hi,

I'm not a web developer (but I have some experience with software) so my terminology/understanding may be wrong...

For the caddy container I have decided to use higher ports instead of the regular ones for the HTTP/HTTPS. This setup worked for me for several months. 
After an update I did less than I week ago I have noticed I no longer can access the web-ui.
Looking at the logs I saw "connection refused" & "timed-out" messages and guessed that the certificate issuer is trying to access some token on the server and just can't reach it. After reverting to regular HTTP/HTTPS ports everything works again.

So... I wouldn't call it a bug - that's why it's not a bug report, but I'd like to move it one day back to these custom ports. Any suggestions how to do it? 

Cheers,
Bartosz

**Comment by dec0dOS** [comment, marked as answer]
Hello, @bartoszbielawski.

Caddy is required to use 80 and 443 ports for Automatic HTTPS feature (ACME challenge) with default settings.
You could try another [option](https://caddyserver.com/docs/automatic-https#acme-challenges), but it will require some Caddy tweaking.

**Reply by bartoszbielawski** [reply]
I see. As far as I can tell the certificate is for the subdomain, there's no port mentioned there, so I have switched to the other port and it works no - probably until the certificate expires in 3 months.
